### PR TITLE
NN-3819: Add Adjudications services

### PR DESCRIPTION
**Pull request by sp-watson**
## What does this pull request do?

Add Adjudications services 

## What is the intent behind these changes?

Make sure the new services are documented

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      }
+
+      val db = system.addContainer("Database", "Storage for adjudication information", "PostgreSQL").apply {
+        Tags.DATABASE.addTo(this)
+      }
+
+      api.uses(db, "connects to")
+    }
+
+    override fun defineRelationships() {
```
Comment (sldblog) [inline src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt:32]: Does this app have any direct users?

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+class AdjudicationsApi private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var api: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "Adjudications API",
+        "Allows reporting and viewing of Adjudications"
```
Comment (sldblog) [inline src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt:18]: Is the API the system itself, or does it sit under a bigger context? (What is the problem space?)

**Review by sldblog — APPROVED** [review]
Thank you! No blockers, just clarifying questions

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt (hunk 1 above)
Comment (sp-watson) [reply src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt:32]: It is just called from the Manage Adjudications app (for now, anyway)

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt (hunk 2 above)
Comment (sp-watson) [reply src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt:18]: w.r.t. Adjudications, this is going to be the single source of truth. But at the moment it manages temporary data that is sent to NOMIS via the Prison API.

**Pull request merged**